ios/brave-ios/Sources/Brave/Frontend/Browser/TabManagerNavDelegate.swift

```diff
@@ -45,6 +45,31 @@ class TabManagerNavDelegate: NSObject, WKNavigationDelegate {
     }
   }
 
+  func webView(
```

Comment:
Let's just return false here and not implement in BVC since we aren't going to do anything different there  anyways